src/components/EditListingLocationPanel/EditListingLocationPanel.js

```diff
@@ -8,7 +8,9 @@ import { ListingLink } from '../../components';
 import { EditListingLocationForm } from '../../forms';
 
 import css from './EditListingLocationPanel.module.css';
-
+const LISTING_TYPE = {
+  equipment: 'equipment'
```

Comment:
This Listing_Type const was Duplicated in many files, why don't we declare it in only 1 constant file and export to other files?